server_test.go

```diff
@@ -1943,6 +1945,25 @@ func (s *Server) MustWriteSeries(database, retentionPolicy string, points []infl
 	return index
 }
 
+func (s *Server) executeQuery(q *influxql.Query, db string, user *influxdb.User) influxdb.Results {
```

Comment:
Function naming could be improved. We've got two functions, both of the same name, but one is exported, and other isn't. It's not clear why one is, and one isn't, of it this was just done to prevent a symbol-naming clash.